packages/labs/src/components/select/select.tsx

```diff
@@ -290,6 +297,13 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
             this.list.scrollActiveItemIntoView();
         }
 
+        requestAnimationFrame(() => {
+            const { inputProps = {} } = this.props;
+            if (inputProps.autoFocus !== false && this.input != null) {
```

Comment:
FYI I had to mull over this logic in my first glance. could use a quick comment like `// autofocus by default` I think.